package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -602,19 +607,50 @@ def _read_next_timestep(self):
             except IndexError:
                 raise ValueError("No coordinate information detected")
         elif not self.lammps_coordinate_convention in convention_to_col_ix:
-            raise ValueError(f"No coordinates following convention {self.lammps_coordinate_convention} found in timestep")
+            raise ValueError("No coordinates following convention"
+                    + f"{self.lammps_coordinate_convention} found in timestep")
 
         coord_cols = convention_to_col_ix[self.lammps_coordinate_convention]
 
         ids = "id" in attr_to_col_ix
+
+        # Create the data arrays for additional attributes which will be saved
+        # under ts.data
+        additional_keys = []
+        if len(attrs) > 3:
+            for attribute_key in attrs:
+                # Skip the normal columns
+                if attribute_key == "id" or \
+                        attribute_key in \
+                        self._coordtype_column_names[
+                        self.lammps_coordinate_convention] \
+                        or attribute_key not in self._additional_columns:
```

Comment:
Correct me if I'm wrong (I'm completely new to MDAnalysis) but I think `self._additional_columns` may not exist when initializing the dump parser with `additional_columns=False` as it is set to default. At least it produces an error in my setup if I don't set `additional_columns` to a  list.

Anyway, very useful Pull request, thanks!

Reply:
Yep, you're right. I have fixed that case.